CMakeLists.txt

```diff
@@ -207,7 +237,10 @@ if (CONFIG_BUILD_WITH_TFM)
     ${TFM_BL2_ARG}
     ${TFM_IPC_ARG}
     ${TFM_REGRESSION_ARG}
+    ${TFM_PSA_TEST_ARG}
   )
 
   zephyr_link_libraries(tfm_api)
+  zephyr_sources(
+    ${ZEPHYR_BASE}/modules/tfm/src/zephyr_tfm_ns_interface.c)
```

Comment:
We are in the TF-M repo but needs a file in the Zephyr repo.

Looking forward to get: https://github.com/zephyrproject-rtos/zephyr/pull/30905 merged.